presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -184,6 +184,11 @@ public void printFinalInfo()
                     (int) percentage(stats.getCpuTimeMillis(), stats.getWallTimeMillis()));
             out.println(cpuTimeSummary);
 
+            // Peak memory: 1.97G
```

Comment:
Since you pass `true` to `formatDataSize` call below the output will be in long form: `1.97GB`, so please update the comment.